be/src/Project/Services/Identity/Identity.Api/v1/Controllers/OrganisationsController.cs

```diff
@@ -39,7 +39,6 @@ public OrganisationsController(IOrganisationService organisationService)
         /// <returns>The seller.</returns>
         [HttpGet, MapToApiVersion("1.0")]
         [Route("{email}")]
-        [AllowAnonymous]
```

Comment:
I would like to force authorization, because without it someone would be able to check whether a given email exists and generally get information about a given organization